EIPS/eip-sc.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: <to be assigned>
+title: Interface for Security Contact of Smart Contract
+description: An interface for security notice using asymmetric encryption
+author: Zainan Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-interface-for-security-contract/10303
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-09
+requires: 165
+---
+
+An interface for smart contract developer to specify to communicate to them in secured way.
+A typical use case is for security researcher to report to developer the
+security vulnerability.
+
+## Abstract
+An interface for security notice using asymmetric encryption. The interface exposes a asymmetric encryption key
+and a destination of delivery.
+
+## Motivation
+When security researchers identify security risk of a smart contract, currently there is no official way to
+report the problem to the smart contract maintainer, such as MakerDAO's issue.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+interface ERC_Security_Contact {
+
+    /// REQUIRED
+    function getSecurityContact(bytes memory data) public view
+    returns (
+        uint8 type,
+        bytes memory publicKey,
+        bytes memory extraData
+    );
+
+    /// OPTIONAL
+    function setSecurityContact(
+        uint8 type,
+        bytes memory publicKey,
+        bytes memory extraData) public;
+    event OnSetSecurityContact(uint8 type, bytes memory publicKeyForEncryption, bytes memory extraData);
+
+    /// OPTIONAL
+    function securityNotify(uint8 type, bytes memory data) public payable;
+    /// OPTIONAL
+    event OnSecurityNotification(uint8 type, bytes memory sourceData, uint256 value);
+
+    /// OPTIONAL
+    function bountyPolicy(uint id) public view returns(string);
```

Comment:
I think this can be left out entirely, and possibly done as a separate EIP.

Reply:
That sounds a good idea. Let me think about it. Added a TODo